backend-models/service-tests/fileedservice-tests.txt

```diff
@@ -121,14 +121,41 @@ if ($result && $_SESSION['uid'] == 1) {
 Test the user's access to a course (line 39)
 --------------------------------------------------------------------------------------------------------------------------------------------
 
+*********
+* pre-req:
+*********
 
+$cid = 5678;
+$userid = 1234;
+$expectedResults = [
+    'w' => false, // should be false if the user does not have 'w' access level
+    'st' => true, // should be true if the user has 'st' access level
+    'sv' => true, // should be true if the user has 'sv' access level
+];
 
-- Test 4: Test the user's access to a course:
-	1. Start up the session.
-	2. Set the user ID in the session.
-	3. Call the function "hasAccess()". Try different access levels.
-	4. Examine the returning values.
-	5. For the test to pass, the returning value should be false if the user does not have access to the course and true if the user has.
+
+*********
+* Start up the session:
+*********
+
+session_start();
+
+
+*********
+* Test access:
```

Comment:
as i stated before this is not a service that was suppose to be tested. as such i'm having a hard time giving useful information directly associated to this commit.

Here you have a complete example of a service test that might be useful. this follows the new standard that is expected.

duggaedservice.php

---------------------------------------------------------------------------------------------------------------------
- Create an assignment (line 57):
---------------------------------------------------------------------------------------------------------------------

*********
* pre-req:
********* 

    { (checklogin) IS TRUE && 
    (hasAccess($userid, $cid, 'w') || isSuperUser($userid) || hasAccess($userid, $cid, 'st')) } IS TRUE


*******
* send parameters inputs to duggaedservice.php 
******
send
{
    $opt = "SAVDUGGA";
    $qid = null; or $qid = “UNK”;
    $cid = 1885;
    $userid = 2;        // this is automatically added depending on what session is active (if any), we want the value to be 2
    $courserver = 1337;

} 

*****
* Get newly creaded Assignment ID
*****
    
    SQL-query: SELECT MAX(id) FROM quiz;     // save to variable $id this value will identify the last assignment that was created (it has the highest ID number) 

*****
* Gather service output
*****

    Save the values of the echoed array as a JSON (at the end of the file. echo json_encode($array);)
    this is the expected output for the micro service.  

*****
* remove inserted row from DB, using qid 
*****

    SQL-query: DELETE FROM quiz WHERE id = $id;